package.json

```diff
@@ -2,9 +2,10 @@
   "name": "leakage",
   "version": "0.2.0",
   "description": "Memory leak testing for node. Write tests using your favorite test runner.",
-  "main": "lib/index.js",
+  "main": "build/index.js",
   "scripts": {
-    "test": "standard lib/**/*.js && mocha test/"
+    "build": "babel lib -d build",
```

Comment:
I think it's more common to have a `src/` and a `lib/` directory. What do you say?